public/js/routes/line/index.js

```diff
@@ -7,62 +10,66 @@ var TripsCollection = require( '../../collections/trips' );
 var LineStatusModel = require( '../../models/line-status' );
 
 var data = require( '../../data' );
-var setTitle = require( '../../lib/set-title' );
 
-function lineOverviewRoute( lineSlug ) {
-  /* jshint validthis: true */
+module.exports = {
+  url: '^/:line(red|orange|blue)',
 
-  // Error out early if the route didn't get a valid line slug
-  var invalidLineSlug = [ 'red', 'orange', 'blue' ].indexOf( lineSlug ) < 0;
+  update: function( opts ) {
+    this.enter( opts );
+  },
```

Comment:
In Ember, it turns out that "update" is the primary hook that is used to fetch data, show the view etc. And "enter" is just a no-op. So, were this Ember, you would just move everything into `update` and probably ignore enter altogether.

It's likely that this cannot be done in Stateman because of the way update moves **up** the routes (like the `exit` hook), rather than down the routes (like `enter`).

I haven't used Stateman myself, so I haven't put any thought into the best way to use its hooks to do common actions like what you're describing.